sormas-backend/src/test/java/de/symeda/sormas/backend/importexport/DatabaseExportServiceTest.java

```diff
@@ -78,18 +93,40 @@ public void test_all_entities_have_export_configuration() {
 				if (!exportableTables.contains(tableName)) {
 					missingEntities.add(clazz.getSimpleName());
 				} else if (NOT_EXPORTED_ENTITIES.contains(clazz.reflect())) {
-					exportedButNotWanted.add(clazz.getSimpleName());
+					exportedButNotWantedEntity.add(clazz.getSimpleName());
+				}
+			}
+
+			for (JavaMethod method : clazz.getMethods()) {
```

Comment:
this for cycle should go inside the if above.

There is no need to search for @JoinTable in not entity classes